restapi/header/header.avsc

```diff
@@ -4,8 +4,13 @@
   "name": "Header",
   "doc": "Metadata for Dataset. It is a summary explaining what kind of information are stored in the Dataset.",
   "fields": [
-    {"name": "descriptiveStatistic", "type": "DescriptiveStatistic" },
-    {"name": "unit", "type": "org.radarcns.avro.restapi.sensor.Unit"},
-    {"name": "effectiveTimeFrame", "type": "EffectiveTimeFrame" }
+    {"name": "subjectId", "type": "string", "doc": "Subject identifier"},
+    {"name": "sourceId", "type": "string", "doc": "Source identifier"},
+    {"name": "source", "type": "org.radarcns.avro.restapi.source.SourceType", "doc": "Source type"},
+    {"name": "sensor", "type": "org.radarcns.avro.restapi.sensor.SensorType", "doc": "Sensor that has generated data"},
```

Comment:
To use this `header` with raw data, I would suggest to make `descriptiveStatistic` and `timeFrame` `nullable`. But to return raw data using `Dataset`, the `Item` schema has to support all raw data schemas and `startDateTime` can be `nullable`. Just to make it clearer
```json
{
  "namespace": "org.radarcns.avro.restapi.dataset",
  "type": "record",
  "name": "Item",
  "doc": "Dataset item. It contains a value and its related time.",
  "fields": [
    {"name": "sample", "type": [
      "org.radarcns.avro.restapi.data.DoubleSample" ,
      "org.radarcns.avro.restapi.data.Acceleration",
      "org.radarcns.empatica.EmpaticaE4Acceleration",
      "org.radarcns.biovotion.BiovotionVSMAcceleration",
      "..."
      ], "doc": "..." } ,
    {"name": "startDateTime", "type": ["null", "string"], "doc": "...", "default": null}
  ]
}
```